Sources/TSCBasic/Process.swift

```diff
@@ -180,10 +180,10 @@ public final class Process {
     // process execution mutable state
     private enum State {
         case idle
-        case readingOutputThread(stdout: Thread, stderr: Thread?)
         case readingOutputPipe(sync: DispatchGroup)
         case outputReady(stdout: Result<[UInt8], Swift.Error>, stderr: Result<[UInt8], Swift.Error>)
         case complete(ProcessResult)
+        case failed(Swift.Error)
```

Comment:
is there a different Error type in this code base that requires qualifying it as Swift.Error?

Reply:
Yes, `ProcessResult.Error`.